frontend/src/metabase/admin/settings/slack/components/SlackSetupForm/SlackSetupForm.tsx

```diff
@@ -1,33 +1,18 @@
-import React, { useMemo } from "react";
-import { t } from "ttag";
-import Form from "metabase/containers/FormikForm";
+import React from "react";
 import { SlackSettings } from "metabase-types/api";
-import { getSlackForm } from "../../forms";
-import { FormProps } from "./types";
-import { FormMessage } from "./SlackSetupForm.styled";
+import SlackForm from "../SlackForm";
+
+const DEFAULT_SETTINGS: SlackSettings = {
+  "slack-app-token": "",
+  "slack-files-channel": "",
```

Comment:
Should these values default to `null` as it represents an empty input? Their types also suggest it could be null
https://github.com/metabase/metabase/pull/26273/files#diff-38a91baa5226c636d59fb05a5529b66f2d66b03bcc95cf3c945f88601bc7e44dR90-R91

Reply:
Nope, they are required form fields. They can be initially `null`